src/Benchmark/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
 cmake_minimum_required(VERSION 3.0)
 
-set(BENCHMARK_SOURCE_FILES
-    Geometry/KDTreeFlann.cpp
-    Geometry/SamplePoints.cpp
-    Core/Reduction.cpp
-)
+file(GLOB_RECURSE BENCHMARK_SOURCE_FILES "*.cpp")
```

Comment:
IMHO, listing out the source files explicitly seems to be preferred, see [this discussion](https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak). 

In practice, I often run into this problem when using glob approach: if a source file is deleted, cmake will fail since it expect the file to be there. This requires re-running the configure step, or sometimes requires a complete clean up of the build directory. 

Since globbing is also used in other parts of Open3D, it is fine to use it here. We can do a clean up of the repository later. What do you think?

Reply:
Lets stick with listing explicitly.  It maybe harder to notice missing .cpp's for UnitTest and Benchmark because "everything still works" (we just don't end up running some of the tests...), maybe at some point we can make a CI test: all .cpp's under UnitTest need to be listed in cmake, and same for Benchmark, so we don't end up accidentally not including some tests